Sources/PinwheelSDK/Classes/Events/PinwheelDDFormCreate.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+public struct PinwheelDDFormCreate: PinwheelEventPayload {
```

Comment:
[nit] : rename to `PinwheelDDFormCreatePayload` or something that'd tell it's an instance of a payload